internal/clients/nightfall/nightfall.go

```diff
@@ -154,13 +159,19 @@ func sliceListBySize(index, numItemsForMaxSize int, contentToScanList []*content
 	return contentToScanList[startIndex:endIndex]
 }
 
-func createCommentsFromScanResp(inputContent []*contentToScan, resp [][]nightfallAPI.ScanResponse, detectorConfigs nightfallconfig.DetectorConfig) []*diffreviewer.Comment {
+func createCommentsFromScanResp(
+	inputContent []*contentToScan,
+	resp [][]nightfallAPI.ScanResponse,
+	detectorConfigs nightfallconfig.DetectorConfig,
+	tokenWhitelist map[string]bool,
+) []*diffreviewer.Comment {
 	comments := []*diffreviewer.Comment{}
 	for j, findingList := range resp {
 		for _, finding := range findingList {
-			if foundSensitiveData(finding, detectorConfigs) {
+			if foundSensitiveData(finding, detectorConfigs) &&
+				!isFindingInTokenWhitelist(finding.Fragment, tokenWhitelist) {
```

Comment:
just to clarify - the `finding.Fragment` is just the sensitive token, no pre or post-context included, right?

Reply:
that's correct